src/components/menu/menu.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from 'react';
+
+import MenuItem from './menu-item';
+export interface Props {
+  pathname: string;
+}
+
+const Menu = ({ pathname }: Props): ReactElement => {
+  const navItems: NavigationItem[] = [
+    {
+      key: 'home',
+      text: 'Home',
+      navigateTo: '/',
+    },
+    {
+      key: 'recoil-example-page-1',
+      text: 'Recoil Example',
+      navigateTo: '/recoil-example-page-1',
+    },
+  ];
+
+  return (
+    <div>
+      <div className="flex space-x-4">
+        {navItems.map((item: NavigationItem) => (
+          <MenuItem
+            id={item.key}
+            active={item.navigateTo === pathname}
+            navigateTo={item.navigateTo}
+            key={item.key}
+            text={item.text}
+            // icon={<item.icon size={16} />}
```

Comment:
Can be removed

Reply:
I left this in here as a wee example of how to add an icon